src/patternfly/components/Form/examples/text-inputs.hbs

```diff
@@ -0,0 +1,53 @@
+{{#>group}}
+  {{#>label labelFor="textInput1"}}Standard input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput1" inputType="text" inputName="textInput1"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput2"}}Input with placeholder{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput2" inputType="text" inputPlaceholder="Example2"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput3" labelStyleModifier="pf-is-required"}}Required input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput3" inputType="text" required=true}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label labelFor="textInput3"}}Disabled input{{/label}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput3" inputType="text" disabled="true"}}{{/text-input}}
+  {{/form-control}}
+{{/group}}
+{{#>group}}
+  {{#>label-group}}
+    {{#>label labelFor="textInput4" labelStyleModifier=" pf-is-required"}}Label with tooltip{{/label}}
+    <div class="pf-c-tooltip">
+      <i class="fas fa-info-circle"></i>
+    </div>
+  {{/label-group}}
+  {{#>form-control}}
+    {{#>text-input inputId="textInput4" inputType="text" required=true}}{{/text-input}}
+  {{/form-control}}
+  {{#>helper-text}}This is helper text{{/helper-text}}
+{{/group}}
+{{#>group}}
+  {{#>label-group}}
+    {{#>label labelFor="textInput5" labelStyleModifier="pf-is-required" inputFor="textInput5"}}Required input with tooltip and error{{/label}}
+    <div class="pf-c-tooltip">
+      <i class="fas fa-info-circle"></i>
+    </div>      
+  {{/label-group}}
+  {{#>form-control controlStyleModifier="pf-has-error"}}
+    {{#>text-input inputStyleModifier="pf-has-error" inputId="textInput5" inputType="text" inputPlaceholder="Has error" required=true}}{{/text-input}}
+  {{/form-control}}
+  {{#>helper-text helperStyleModifier="pf-has-error"}}This is helper text{{/helper-text}}
```

Comment:
For this example, I'd suggest using `aria-errormessage` instead of `aria-describedby`. This also requires the use of `aria-invalid` which is noted here: https://www.w3.org/TR/wai-aria-1.1/#aria-errormessage. I also think we should test this pattern using `aria-live`, but I think it would be better to flush that out when working on the JS implementation where it would be easier to test.